File: CH3-desicion-sructure-and-boolean-logic/EX4-magic-dates/source-code/magicDates.cpp
#include <iostream>
using namespace std;

int main()
{
  int month;
  int day;
  int year;
  int result;

  cout << "Enter a month :";
  cin >> month;

  cout << "Enter a day   :";
  cin >> day;

  cout << "Enter a year  :";
  cin >> year;

  result = month * day;

  if (result == year)
  {
    cout << "the date is magic";
  }
  else
  {
    cout << "the date is not magic";
  }
}

File: CH3-desicion-sructure-and-boolean-logic/EX4-magic-dates/source-code/magicDates.php
<?php

$month;
$day;
$year;
$result;

$month = readline("Enter a month :");
$day   = readline("Enter a day   :");
$year  = readline("Enter a year  :");

$result = $month * $day;

if($result == $year){
  echo "the date is magic";
}else{
  echo "the date is not magic";
}
